internal/connect/command_offset_describe.go

```diff
@@ -71,36 +70,22 @@ func (c *offsetCommand) getOffset(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	connectorIdToName, err := c.mapConnectorIdToName(environmentId, kafkaCluster.ID)
+	connector, err := c.V2Client.GetConnectorExpansionById(args[0], environmentId, kafkaCluster.ID)
 	if err != nil {
 		return err
 	}
 
-	offsets, err := c.V2Client.GetConnectorOffset(connectorIdToName[args[0]], environmentId, kafkaCluster.ID)
+	connectorName := connector.Info.GetName()
+	offsets, err := c.V2Client.GetConnectorOffset(connector.Info.GetName(), environmentId, kafkaCluster.ID)
 	if err != nil {
 		return err
 	}
 
 	if output.GetFormat(cmd) == output.Human {
-		return printHumanDescribeOffset(cmd, offsets, args[0], connectorIdToName[args[0]])
+		return printHumanDescribeOffset(cmd, offsets, args[0], connectorName)
 	}
 
-	return printSerializedDescribeOffsets(cmd, offsets, args[0], connectorIdToName[args[0]])
-}
-
-func (c *offsetCommand) mapConnectorIdToName(environmentId, kafkaClusterId string) (map[string]string, error) {
-	// NOTE: Do NOT replace this with `V2Client.GetConnectorExpansionById` calls; that function itself calls `V2Client.ListConnectorsWithExpansions`
-	connectors, err := c.V2Client.ListConnectorsWithExpansions(environmentId, kafkaClusterId, "id,info,status")
-	if err != nil {
-		return nil, err
-	}
-
-	connectorIdToName := make(map[string]string)
-	for _, connector := range connectors {
-		connectorIdToName[connector.Id.GetId()] = connector.Info.GetName()
-	}
-
-	return connectorIdToName, nil
+	return printSerializedDescribeOffsets(cmd, offsets, args[0], connectorName)
```

Comment:
Please combine into one function (one call to `table.Print()`). See other functions for an example.